### PR TITLE
Add the links to the figures in the documentation of GridGenerator::eccentric_hyper_shell()

### PR DESCRIPTION
**Pull request by melaniegerault**

### CODE REVIEW

File: include/deal.II/grid/grid_generator.h [hunk 1]
```diff
@@ -977,8 +977,15 @@ namespace GridGenerator
    * Here, the number <tt>n_cells</tt> of elements has the same meaning as in
    * GridGenerator::hyper_shell.
    *
-   * @note This function is declared to exist for triangulations of all space
-   * dimensions, but throws an error if called in 1d.
+   * The grids with 12 initial cells and 3 levels of global refinement are
+   * plotted below:
+   *
+   * @image html eccentric_hyper_shell_2D.png
+   * @image html eccentric_hyper_shell_2D.png
```
Comment (drwells) [inline include/deal.II/grid/grid_generator.h]: I think that you meant to use `eccentric_hyper_shell_3D.png` here.

**Comment by drwells** [comment]
I don't think it matters too much how we solve this, but the grid pictures are way too large: they are about 4x the size of the images in `hyper_shell`, for example. It would also be nice if we trimmed the whitespace a bit.

Do you have imagemagick installed? If so, its easy to resize images:
```shell
> convert -resize 50% eccentric_hyper_shell_2D.png eccentric_hyper_shell_2D.png
```
Let me know if you need help with resizing the pictures.

**Comment by bangerth** [comment]
And `convert -trim` removes the white frame.

**Comment by melaniegerault** [comment]
I use imageMagick too! Thanks, will do.

Mélanie

On Aug 14, 2019, at 3:53 PM, Wolfgang Bangerth <notifications@github.com<mailto:notifications@github.com>> wrote:


And convert -trim removes the white frame.

File: include/deal.II/grid/grid_generator.h (hunk 1 above)
Comment (melaniegerault) [reply include/deal.II/grid/grid_generator.h]: Yes, thanks!

**Comment by melaniegerault** [comment]
Hopefully this last commit solves these issues.
Thanks a lot @drwells @bangerth and @tamiko !

**Comment by drwells** [comment]
The pictures are resized and trimmed; I generated the doxygen and it looks nice. Thanks again @melaniegerault!

**Comment by melaniegerault** [comment]
Well I should thank you instead @drwells ! :)

**Pull request merged**